rubicon/objc/runtime.py

```diff
@@ -888,6 +904,50 @@ def add_ivar(cls, name, vartype):
     )
 
 
+def get_weak_ivar(obj, varname):
```

Comment:
Would it make sense to merge this into the existing `get_ivar` function (using a `weak` keyword argument that defaults to `False`) instead of defining a separate function? As far as I can tell, the code in `get_weak_ivar` and `get_ivar` is very similar.

The same question applies to `set_weak_ivar`/`set_ivar`.

Reply:
Yes, I think this makes sense.